ruma-federation-api/src/identification.rs

```diff
@@ -0,0 +1,3 @@
+//! Module for dealing with third party identifiers
+
+pub mod identifier_bound;
```

Comment:
In order to stick closer to the organization of the Federation API, I wonder if we should use `thirdparty` for the module name and call the endpoint `bind_callback` or something.

At the same time, the only other endpoint in that spec module, 3PID invitation, would probably fit better in the `membership` module here. On top of that, in `ruma-client-api`, we smoosh all the 3PID management stuff under `account`. Soooo.... I don't know, haha. What do you guys think?

Reply:
Endpoint grouping is hard and I don't have the time to understand the Federation API in enough detail to provide suggestions here, sorry.

Reply:
Yeah this one was hard to come up with module names for. I think I prefer @iinuwa's first suggestion though.